### PR TITLE
JES authentication documentation

### PR DESCRIPTION
**Pull request by scottfrazer**
I did not add documentation for the user authentication portion because I didn't understand how it works!  If somebody would like to explain it to me I can expand these docs

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -506,16 +510,31 @@ Since the `script.sh` ends with `echo $? > rc`, the backend will wait for the ex
 
 Google JES (Job Execution Service) is a Docker-as-a-service from Google. JES has some [configuration](#configuring-cromwell) that needs to be set before it can be run.  Edit `src/main/resources/application.conf` and fill out the 'jes' stanza, e.g.
```
Comment (cjllanwarne) [inline README.md]: This line refers to a blocks which is now ~15 lines further away

File: README.md [hunk 2]
```diff
@@ -524,30 +543,71 @@ backend {
...
+
+```hocon
+google {
+  applicationName = "cromwell"
+  cromwellAuthenticationScheme = "application_default"
 }
 ```
 
-It is also necessary to fill out the `google` stanza in the configuration file. This stanza will set up the service / user account that Cromwell uses to write certain files to GCS as well as run jobs.
+To authenticate, run the following commands:
```
Comment (cjllanwarne) [inline README.md]: "... on your local command line"

File: README.md [hunk 3]
```diff
@@ -524,30 +543,71 @@ backend {
...
+
+```
+$ gcloud auth login
+$ gcloud config set project my-project
+```
+
+This should be all that's necessary to run Cromwell using the JES backend.
+
+#### Service Account
```
Comment (cjllanwarne) [inline README.md]: Do you not need to create a service account before you do the gcloud auth login?

Comment (scottfrazer) [inline README.md]: I'm not sure!  I'll look into it

**Comment by cjllanwarne** [comment]
:+1:

File: README.md (hunk 3 above)
Comment (Horneth) [inline README.md]: No you don't, when you gcloud auth login you login using a gmail account basically. You can still [use gcloud to authenticate using a service account](https://cloud.google.com/sdk/gcloud/reference/auth/activate-service-account) but you don't have too

**Comment by Horneth** [comment]
:+1: 

[Approved with PullApprove](https://pullapprove.com/broadinstitute/cromwell/pull-request/532/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=broadinstitute/cromwell)

**Pull request merged**